rust/connlib/libs/common/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "libs-common"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+jni-bindings = ["boringtun/jni-bindings"]
+
+[dependencies]
+base64 = { version = "0.21", default-features = false, features = ["std"] }
+serde = { version = "1.0", default-features = false, features = ["derive", "std"] }
+futures =  { version = "0.3", default-features = false, features = ["std", "async-await", "executor"] }
+futures-util =  { version = "0.3", default-features = false, features = ["std", "async-await", "async-await-macro"] }
+tokio-tungstenite = { version = "0.18", default-features = false, features = ["connect", "handshake"] }
+webrtc = { version = "0.7" }
+uuid = { version = "1.3", default-features = false, features = ["std", "v4", "serde"] }
+thiserror = { version = "1.0", default-features = false }
+tracing = { version = "0.1", default-features = false, features = ["std", "attributes"] }
+serde_json = { version = "1.0", default-features = false, features = ["std"] }
+tokio = { version = "1.28", default-features = false, features = ["rt", "rt-multi-thread"]}
+url = { version = "2.3.1", default-features = false }
+rand_core = { version = "0.6.4", default-features = false, features = ["std"] }
+async-trait = { version = "0.1", default-features = false }
+backoff = { version = "0.4", default-features = false }
+boringtun = { git = "https://github.com/cloudflare/boringtun", rev = "878385f", default-features = false }
+ip_network = { version = "0.4", default-features = false, features = ["serde"] }
+
+macros = { path = "../../macros" }
+
+[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
+swift-bridge = { git = "https://github.com/conectado/swift-bridge.git", branch = "fix-already-declared" }
```

Comment:
Could also specify this as a workspace dependency, see https://github.com/libp2p/rust-libp2p/blob/2c6c058bb7b0d373e05835fead3f8e63e3240583/Cargo.toml#L62-L103 for how to define that.